docs/pipeline_config.md

```diff
@@ -0,0 +1,14 @@
+# Pipeline Configuration
+
+Built-in pipeline configurations can be found in [`src/instructlab/sdg/pipelines/`](../src/instructlab/sdg/pipelines/).
+
+## Pipeline Configuration Schema
+
+A schema for validating pipeline configuration can be found in [`src/instructlab/sdg/pipelines/schema/v1.json`](../src//instructlab/sdg/pipelines/schema/v1.json)
+
+## Version History
+
+| Version | Description |
+| ---     | --- |
+| 1.0     | Initial version |
+| 1.1     | Added `model_prompt` to `config` for `LLMBlock` |
```

Comment:
FWIW, I'd be happy enough for us to keep changing 1.0 until we release it for a first time?

Reply:
Yeah, I'm fine with that.

I kinda got carried away thinking about how we'd document changes.  I'm totally fine pulling this from the PR for now and coming back to it later.

Reply:
I rebased and pulled the version bump out. I left the doc in place though with a link to the schema and a version table that just lists 1.0.